tests/class properties with super/options.json

```diff
@@ -0,0 +1,4 @@
+{
+	"supportedBabels": ["babel 7"],
```

Comment:
The output here seems to differ depending on the Babel version – seems like that could be handled by having a separate test case for Babel 6 but not sure if that's how this is typically done.